lib/internal/fs/streams.js

```diff
@@ -132,6 +132,16 @@ ReadStream.prototype.open = function() {
     }
 
     this.fd = fd;
+
+    if (this.destroyed) {
+      closeFsStream(this, (er) => {
+        if (er) {
+          this.emit('error', er);
+        }
```

Comment:
This will cause error to be emitted after close. I think we should swallow this error.

Reply:
I removed it and the error still got emitted. The `error` event does not seem to come from lib/internal/fs... 🤔

```
Path: parallel/test-fs-stream-no-events-after-destroy
assert.js:128
  throw err;
  ^

AssertionError [ERR_ASSERTION]: function should not have been called at /Users/hackintosh/Documents/code/node/test/parallel/test-fs-stream-no-events-after-destroy.js:15
    at ReadStream.mustNotCall (/Users/hackintosh/Documents/code/node/test/common/index.js:404:12)
    at ReadStream.emit (events.js:189:13)
    at emitErrorNT (internal/streams/destroy.js:82:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:50:3)
    at internalTickCallback (internal/process/next_tick.js:72:19)
    at process._tickCallback (internal/process/next_tick.js:47:5)
    at Function.Module.runMain (internal/modules/cjs/loader.js:778:11)
    at startup (internal/bootstrap/node.js:300:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:826:3)
```